programs/management/data/tx_csfp_initial_config.json

```diff
@@ -0,0 +1,56 @@
+{
+    "white_label": {"code": "tx"},
+    "program_category": {"external_name": "tx_food", "name": "Food and Nutrition", "icon": "food"},
+    "program": {
+        "name_abbreviated": "tx_csfp",
+        "year": "2025",
+        "legal_status_required": [
+            "citizen",
+            "non_citizen",
+            "gc_5plus",
+            "gc_18plus_no5",
+            "gc_under18_no5",
+            "otherWithWorkPermission"
+        ],
+        "name": "Commodity Supplemental Food Program",
```

Comment:
Should we add `(CSFP)` to the title? I see the learn more link below uses both. I don't know enough about the program to know whether folks in the community would recognize either